build.py

```diff
@@ -15,8 +17,23 @@ def test(*args):
     """
     Run unit tests.
     """
-    subprocess.call(["py.test-2.7"] + list(args))
-    subprocess.call(["py.test-3.3"] + list(args))
+    try:
+        subprocess.call(["py.test-2.7"] + list(args))
+    except FileNotFoundError:
+        print("py.test-2.7 not found, trying generic name")
+    try:
+        subprocess.call(["py.test"] + list(args))
```

Comment:
The idea was to use python 2 and 3 here to make sure its compatiable with both. From what I understand "py.test" could be either so it doesnt enforce python 2?